src/App.ts

```diff
@@ -1624,8 +1624,8 @@ function extractFunctionContext(body: StringIndexed) {
 
   // interactivity (block_actions)
   if (body.function_data) {
-    functionExecutionId = body.function_data.execution_id;
-    functionBotAccessToken = body.bot_access_token;
+    functionExecutionId = body.function_data.execution_id || functionExecutionId;
```

Comment:
I don't follow the logic here: AFAIK, the two conditions never would happen in one go?